frontend/openchat-client/src/openchat.ts

```diff
@@ -5180,23 +5196,35 @@ export class OpenChat extends OpenChatAgentWorker {
     }
 
     private async updateRegistry() {
-        const registry = await this.sendRequest({
+        const [registry, updated] = await this.sendRequest({
             kind: "updateRegistry",
         });
 
-        const cryptoRecord = toRecord(registry.tokenDetails, (t) => t.ledger);
+        if (updated) {
```

Comment:
I think this will fail on first load, because the stores won't have been set, but the registry won't have had any updates.

Reply:
Maybe pass in an `initialLoad` flag or something like that